src/common/common.css

```diff
@@ -171,6 +171,20 @@ a:active {
   margin-top: 8px;
 }
 
+#messageContrast {
```

Comment:
Hmm `#messageContrast`  is only one case where this might happen.
It would be best if the solution were message-agnostic, i.e. somehow integrate it into `.message-box`.